crates/bevy_audio/Cargo.toml

```diff
@@ -18,11 +18,14 @@ bevy_utils = { path = "../bevy_utils", version = "0.6.0" }
 
 # other
 anyhow = "1.0.4"
-rodio = { version = "0.14", default-features = false }
+rodio = { version = "0.15", default-features = false }
+# we do not directly depend on cpal, but we require at least this version to be used by rodio
+# so that the winit feature of drag and drop can be enabled https://github.com/RustAudio/cpal/issues/572
+cpal = ">=0.13.4"
```

Comment:
I am not very happy with adding this.

Not adding it would mean that some users would have an issue until they run `cargo update`.
Those users would be those:
* targeting windows
* with a lock file not updated since cpal 0.13.4 has been updated (on the 21/08/06 https://crates.io/crates/cpal/0.13.4)
* with Bevy after the 0.6

I think this set of users is small enough to not add this line to the file

Reply:
Yeah, that's reasonable. Will just bumping `rodio` fix this?

Reply:
no, rodio merely requires `cpal` to be `0.13`, be it the currently used one or the new version, so it will stick to the old version recorded in the cargo.lock

I figured adding this requirement for a few weeks or so would be a good idea, as that will not cause people's builds to randomly break. Kind of like a transition period for the people that use bevy from the `main` branch instead of the crates.io version.

Reply:
IMO we should just remove this. Breakage is much less of a concern than tech debt for this project.